python/parpe/loadleveler.py

```diff
@@ -0,0 +1,54 @@
+from jinja2 import Template, Environment, PackageLoader
+
+env = Environment(loader=PackageLoader('parpe', 'templates'))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [By default, jinja2 sets autoescape to False. Consider using autoescape=True or use the select_autoescape function to mitigate XSS vulnerabilities.](https://app.codacy.com/app/dweindl/parPE/pullRequest?prid=3660516)